lighthouse-core/gather/gather-runner.js

```diff
@@ -207,7 +187,6 @@ class GatherRunner {
           log.log('status', 'Disconnecting from browser...');
           driver.disconnect();
```

Comment:
While you're here can you reformat the chaining of these promises? (reloadForCleanStateIfNeeded & disconncet) Like, pull the disconnect into the parent chain.